Tests/SwiftDocCTests/Diagnostics/DiagnosticTests.swift

```diff
@@ -140,6 +141,52 @@ class DiagnosticTests: XCTestCase {
         \(expectedNoteLocation): note: The message of the note.
         """)
     }
+
+    func createTestSymbol(commentText: String) -> SymbolGraph.Symbol {
+         let emptyRange = SymbolGraph.LineList.SourceRange(
+             start: .init(line: 0, character: 0),
+             end: .init(line: 0, character: 0)
+         )
+        let docCommentLines = commentText.components(separatedBy: .newlines).map { SymbolGraph.LineList.Line(text: $0, range: emptyRange) }
+        return SymbolGraph.Symbol(
+            identifier: .init(precise: "s:5MyKit0A5ClassC10myFunctionyyF", interfaceLanguage: "occ"),
+            names: .init(title: "", navigator: nil, subHeading: nil, prose: nil),
+            pathComponents: ["path", "to", "my file.h"],
+            docComment: SymbolGraph.LineList(docCommentLines),
+            accessLevel: .init(rawValue: "public"),
+            kind: .init(parsedIdentifier: .func, displayName: "myFunction"),
+            mixins: [:]
+        )
+    }
+
+    func testDoxygenDiagnostic() throws {
+        let commentText = """
+          /// Brief description of this method
```

Comment:
Just to avoid confusion for future readers of this text I think we should exclude the triple-slash here. We would expect whatever tool generated the symbol graph to automatically strip any comment delimiters before passing them off to docc.

```suggestion
          Brief description of this method
```